trin-history/src/jsonrpc.rs

```diff
@@ -117,6 +117,9 @@ impl HistoryRequestHandler {
                     };
                     let _ = request.resp.send(response);
                 }
+                HistoryEndpoint::RecursiveFindNodes(node_id) => {
+                    // TODO.... where do I dispatch this...
```

Comment:
This is where I got stuck.  Anyone wanna point me to the right entry point for where to dispatch RFN?